commit/report.go

```diff
@@ -45,7 +50,18 @@ func (p *Plugin) Reports(seqNr uint64, outcomeBytes ocr3types.Outcome) ([]ocr3ty
 		return nil, fmt.Errorf("encode commit plugin report: %w", err)
 	}
 
-	return []ocr3types.ReportWithInfo[[]byte]{{Report: encodedReport, Info: nil}}, nil
+	// Prepare the info data
+	reportInfo := ReportInfo{
+		MinSigners: outcome.MerkleRootOutcome.RMNRemoteCfg.MinSigners,
+	}
+
+	// Serialize reportInfo to []byte
+	infoBytes, err := json.Marshal(reportInfo)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal report info: %w", err)
+	}
+
+	return []ocr3types.ReportWithInfo[[]byte]{{Report: encodedReport, Info: infoBytes}}, nil
```

Comment:
Leverage the Info to access the `MinSigners` in `ShouldAcceptAttestedReport`

Reply:
I checked with research team and it seems like this use case fits the usage the `ReportWithInfo`